stdlib/format.mli

```diff
@@ -1120,6 +1120,12 @@ val pp_print_result :
 
     @since 4.08 *)
 
+val pp_print_either :
+  left:(formatter -> 'a -> unit) ->
+  right:(formatter -> 'b -> unit) -> formatter -> ('a, 'b) Either.t -> unit
+(** [pp_print_either ~left ~right ppf e] prints [e] on [ppf] using
+    [left] if [e] is [Either.Left _] and [right] if [e] is [Either.Right _]. *)
```

Comment:
As for `pp_print_result` I suggest you to add a `@since` anotation.

Reply:
Would that be the next release version, `4.13`?

Reply:
To be honnest, I don't know. According to https://github.com/ocaml/ocaml/pull/10352/files
it seems that `4.13` is good but @gasche or @Octachron can give a better answer.

Reply:
The next version is `4.13` indeed.